pkg/vm/vm_ops_maths.go

```diff
@@ -42,6 +44,56 @@ func Sub(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
 	return NONE, nil
 }
 
+// Inc increments the stack Item's value of 1.
+// Returns an error if the item cannot be casted to an integer
+// or if 1 cannot be added to the item
+func Inc(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rstack *stack.RandomAccess) (Vmstate, error) {
+
+	i, err := ctx.Estack.PopInt()
+	if err != nil {
+		return FAULT, err
+	}
+
+	one, err := stack.NewInt(big.NewInt(1))
+	if err != nil {
+		return FAULT, err
+	}
+
+	res, err := i.Add(one)
+	if err != nil {
+		return FAULT, err
+	}
+
+	ctx.Estack.Push(res)
+
+	return NONE, nil
+}
+
+// Dec decrements the stack Item's value of 1.
```

Comment:
same here, but apart from that looks perfect

Reply:
:+1: